README.md

```diff
@@ -843,13 +843,45 @@ Params:
 - `steps`: a list of steps/actions to execute in the scope of the for_each loop.
   It's analogous to the `steps` field at the top level of the spec file.
 
+### Ignore (Optional)
+
+We use [filepath Glob](https://pkg.go.dev/path/filepath#Glob) to match the file
+and directory paths that should be ignored if included/copied from template
+directory to destination directory, and should not be overwritten if included
+from destination. This `ignore` feature is similiar to `skip` in `include`
+action, the difference here is that ignore is global and it applies to every
+`include` action.
+
+This section is optional, if not provided, a default ignore list is used:
+`.DS_Store`, `.bin`, and `.ssh`. To set your custom ignore list, please check
+accepted patterns [here](https://pkg.go.dev/path/filepath#Match).
+
+Example:
+
+```yaml
+ignore:
+  - '*/*.ssh'
```

Comment:
I think we want `*/.ssh` rather than `*/*.ssh`. The intention with ignoring these is that we want to ignore the specific directory name `.ssh`, and not ignore all files ending with `.ssh`.

Reply:
This is an example to add custom ignore list, I will use `*/*.txt` instead since it is confusing.

Reply:
I see, +1 to using .txt instead